src/frontend/_layouts/app/NavigationSideBar/SideBar.tsx

```diff
@@ -49,6 +50,19 @@ export function SideBar({ isFullWidth, setIsFullWidth }: IProps) {
     setActiveItem$1(newValueFiltered);
   };
 
+  const handleKeyPress = (event: KeyboardEvent) => {
+    if ((event.ctrlKey || event.metaKey) && event.keyCode === 66) {
+      setIsFullWidth(!isFullWidth);
+    }
+  };
+
+  useEffect(() => {
+    document.addEventListener("keydown", handleKeyPress);
+
+    return () => {
+      document.removeEventListener("keydown", handleKeyPress);
+    };
+  }, [handleKeyPress]);
```

Comment:
This will be problematic as the event listener will be re-attached on every re-render, Can you either remove this dependency or use a useCallback for the `handleKeyPress`